src/System.IO.Compression.Brotli/System/IO/Compression/Brotli.cs

```
@@ -200,7 +221,14 @@ public static TransformationStatus Compress(ReadOnlySpan<byte> source, Span<byte
 
         public static TransformationStatus Decompress(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, ref State state)
         {
-            EnsureInitialized(ref state, false);
+            try
```

Same here. Just let this bubble up to the caller.